public/components/main/main_utils.tsx

```diff
@@ -264,3 +272,46 @@ export const generateReportById = async (
       }
     });
 };
+export const sendTestNotificationsMessage = async (
+  id: string,
+  httpClientProps: HttpSetup,
+  item: any
```

Comment:
Try to typed this prop

Reply:
This comes from a prop that already comes from the reporting repository and has as type any